incident/api/views.py

```diff
@@ -1,14 +1,23 @@
 from rest_framework import viewsets
 from rest_framework.settings import api_settings
-from rest_framework_csv.renderers import CSVRenderer
+from rest_framework.pagination import CursorPagination
+from rest_framework_csv.renderers import PaginatedCSVRenderer
 
 from incident.api.serializers import IncidentSerializer, FlatIncidentSerializer
 from incident.utils.incident_filter import IncidentFilter
 
 
+class IncidentCursorPagination(CursorPagination):
+    page_size = 25
+    page_size_query_param = 'limit'
+    max_page_size = 200
```

Comment:
Is it possible to set no maximum? I want API consumers to be able to get the whole database in one request if they *want* but I also want them to have the option of not getting the whole database.

Reply:
Yes, I think that's totally possible! I wasn't sure what our wants/needs were exactly in this regard so I just made something up here.